src/patients/related-persons/RelatedPersonTab.tsx

```diff
@@ -88,14 +88,17 @@ const RelatedPersonTab = (props: Props) => {
         <div className="col-md-12">
           <Panel title={t('patient.relatedPersons.label')} color="primary" collapsible>
             {relatedPersons ? (
-              <List>
-                {relatedPersons.map((r) => (
-                  <ListItem key={r.id}>{r.fullName}</ListItem>
-                ))}
-              </List>
-            ) : (
-              <h1>Loading...</h1>
-            )}
+              (relatedPersons.length > 0) ? (
+                <List>
+                  {relatedPersons.map((r) => (
+                    <ListItem key={r.id}>{r.fullName}</ListItem>
+                  ))}
+                </List>
+                ) : (
+                  <h2>No related persons have been added yet.</h2>
```

Comment:
I think we can use the alert component here, using the warning color (or maybe info)

Reply:
Sounds good! I've made another commit, is this any better?

![related](https://user-images.githubusercontent.com/8657895/74022071-b0e4e600-4994-11ea-80d6-4c248e93f6ee.png)